src/backend/EdFi.DataManagementService.Backend.Postgresql/PostgresqlDocumentStoreRepository.cs

```diff
@@ -49,9 +49,9 @@ public async Task<UpsertResult> UpsertDocument(IUpsertRequest upsertRequest)
             _logger.LogError("Unknown Error - {TraceId}", upsertRequest.TraceId);
             return new UpsertResult.UnknownFailure("Unknown Failure");
         }
-        catch
+        catch (Exception ex)
         {
-            _logger.LogError("Unknown Error - {TraceId}", upsertRequest.TraceId);
+            _logger.LogError(ex, "Unknown Error - {TraceId}", upsertRequest.TraceId);
```

Comment:
Previously the exception was trapped and not logged. SonarLint now detects that situation.